src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/VisitVisitor.kt

```diff
@@ -27,16 +27,15 @@ data class VisitVisitor(
   @Column(name = "OFFENDER_VISIT_VISITOR_ID", nullable = false)
   val id: Long? = null,
 
```

Comment:
mapping the id makes more sense in an insert scenario

Reply:
Do you mean `personId` ? Wouldn't you look up Person anyway to verify the personId passed in is correct?

Reply:
Ah good point, ill do that